brianlong/www.validators.app
# 179251127_add_epoch_credits_to_api_outputs

#### What's this PR do?
- Add epoch credits to api outputs

#### How should this be manually tested?
- Use postman to get validators aor single validator from our API
- Check if you can see `epoch_credits` field

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/179251127

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
